src/components/CardItem/CardItem.tsx

```diff
@@ -0,0 +1,106 @@
+'use client';
+
+import { Typography, Box } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface CardItemInterface {
+  item: {
+    id: string;
+    imageSrc: string;
+    title: string;
+    text: string;
+  };
+  type: 'WhyUs' | 'Possibilities';
+}
+
+function CardItem({ item, type }: CardItemInterface) {
+  const { t } = useTranslation();
+
+  return (
+    <Box>
+      {type === 'Possibilities' && (
+        <Box
+          maxWidth="18.063rem"
+          sx={{
+            height: '100%',
```

Comment:
create a new scss file where you will add all these styles, this approach with the sx property is not good for us, it makes the code more complicated